tests/integration/models/test_peagle_forward.py

```
@@ -0,0 +1,164 @@
+        t2d = torch.zeros(VOCAB_SIZE, dtype=torch.bool)
+        t2d[:32] = True
+        d2t = torch.arange(32, dtype=torch.long)
+        model.load_vocab_mappings(t2d.to("cuda"), d2t.to("cuda"))
+        yield model
+        del model
+        torch.cuda.empty_cache()
+
+    def test_boundary_tokens(self, peagle_draft_vocab_model):
+        samples = _make_samples([128], vocab_size=32, boundary_token_ids=[0, 31])
+        batch = make_batch(max_len=MAX_LEN, samples=samples, hidden_size=HIDDEN_SIZE)
+        draft_tokens, loss, metrics = peagle_draft_vocab_model(**batch)
```

Ruff (0.15.14)

[warning] Unpacked variable `draft_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)